CHANGES

```diff
@@ -47,6 +47,14 @@ standard support.
     Similar to proof-action-list, but holding actions that need
     to go to the proof assistant at the next opportunity.
 
+** Qrhl-tool
+
+   Support for qrhl-tool theorem prover has been added by Dominique
+   Unruh.
+   References:
+   - Initial pull request https://github.com/ProofGeneral/PG/pull/636.
+   - Qrhl-tool web site: https://dominique-unruh.github.io/qrhl-tool.
```

Comment:
Thanks @Matafou !

Detail: I'd suggest we don't put a dot at the end of the URL in raw-text files (unless we add surrounding `<…>`):
```suggestion
   - Initial pull request: <https://github.com/ProofGeneral/PG/pull/636>.
   - Qrhl-tool web site: <https://dominique-unruh.github.io/qrhl-tool>.
```